### DISCUSSION TITLE
Change S2 discount to same values as S1 discount

### DISCUSSION THREAD

**Discussion by minthes** [chromiecraft/chromiecraft#7448] [Suggestions]
Players were expecting the discount values in S2 to match those in S1 for consistency. To meet these expectations and avoid any disappointment, I suggest adjusting S2 discounts to the same values as S1.

**Comment by Nyeriah** [comment]
We are working on implementing a solution to the discount issue, so we are able to grant you the 50% discounts you all have been expecting. All points spent as well as enchants will be refunded, so you don't have to worry about that. It won't come with the next restart though, as it requires some work still.

**Reply by andovcr** [reply]
Yesterday (Oct 25th) an announcement was made that the server would be restarted today (Oct 26th) which would add an NPC to refund Arena Points spent earlier in the week and to fix S2 Arena gear discount prices.

The server was restarted today, but I cannot find the NPC, Rizzy, anywhere in Area 52.  Furthermore several items are now missing from the S2 Arena vendor.  This "update" appears to have not met expectations.

Why?

**Reply by Nyeriah** [reply]
This "update" didn't meet your "expectations" because there were issues that are being worked on.

**Reply by andovcr** [reply]
One suggestion would be to make an announcement of that then. The last official statement on the subject is just blatantly wrong. Players would be more understanding of work that takes time if that is communicated - an announcement takes very little time and effort.

A second suggestion would be to load changes like these onto the PTR first to verify that they actually work before making an announcement that they're live.

**Reply by Nyeriah** [reply]
Thanks for the words of counsel, Andov. Supportive players like you are what really motivates us to keep working on things.
